bitnami/fluentd/templates/forwarder-daemonset.yaml

```
@@ -3,6 +3,10 @@ apiVersion: apps/v1
 kind: DaemonSet
 metadata:
   name: {{ include "fluentd.fullname" . }}
+  {{- if .Values.forwarder.annotations }}
+  annotations:
+    {{- include "fluentd.tplValue" (dict "value" .Values.forwarder.annotations "context" $) | nindent 4 }}
```

```suggestion
  annotations: {{- include "fluentd.tplValue" (dict "value" .Values.forwarder.annotations "context" $) | nindent 4 }}
```
Just for style's sake, this can be only one line as we are already using `nindent`.